### DISCUSSION TITLE
articles/after-git

### DISCUSSION THREAD

**Discussion by giscus** [hutusi/hutusi.github.com#186] [Comments]
# articles/after-git

19世纪末，弗雷德里克·泰勒在美国一些工厂推广他的科学管理方法，这包括用科学的方法来代替人工经验判断、对工人进行专业训练以及按专业分工。亨利·福特在他的汽车工厂里也开始应用泰勒科学管理，并且在汽车生产上创新性的采用了流水线作业。1908年，第一辆福特T型车从流水线上面世，而经过优化的流水线在几年后甚至可以达到93分钟内生产一部汽车，强大的生产效率让福特超过了其他所有汽车厂商生产能力的总和。福特的流水线生产很快被其他厂商所借鉴，也迅速普及到了其他工业领域。可以说，福特T型车流水线作业的发明开启了大规模工业生产的时代，人类的生产力得到了巨大的提升。从英国移居到美国的阿道司·赫胥黎对此深受启发，在他的带有科幻色彩的反乌托邦小说《美丽新世界》中，描写未来的人类“文明社会”，人们不再用公元纪年，也不信仰上帝，改尊亨利·福特为唯一的神，以福特为纪元，并以第一辆福特T型车流水线生产的1908年为福特元年。

https://hutusi.com/articles/after-git

**Comment by Linusp** [comment]
想远一点的话，如果算力足够也有足够多平台支持了，我想 AI 的协作开发（基于 AI 实现各种功能，这个可能已经和我们所说的软件开发不太一样了）可能会更接近维基百科，大家要做的是协同进行知识、逻辑的编辑。不管是 Git 还是维基百科，都是很棒的！

**Reply by hutusi** [reply]
未来可能是算力的天下，人类的知识什么的可能都不重要了：）

**Comment by miao1007** [comment]
lfs下载有一个缺陷，git lfs fetch后会占用两份二进制空间，下模型还是挺心疼的

**Reply by hutusi** [reply]
确实，不过不用的话是不是可以用 `git lfs prune` 给删掉~
